src/SegmentedMsg.chpl

```diff
@@ -489,6 +489,38 @@ module SegmentedMsg {
     return new MsgTuple(repMsg, MsgType.NORMAL);
   }
 
+  proc segmentedStripMsg(cmd: string, payload: string, st: borrowed SymTab): MsgTuple throws {
+    var pn = Reflection.getRoutineName();
+    var repMsg: string;
+
+    var (subcmd, objtype, name, containsSpace, pattern) = payload.splitMsgToTuple(5);
```

Comment:
It would probably be ideal to update the parsing here so that pattern can include spaces. You can limit the number of splits done on payload to do this. I think this would be better than adding the check for contains space.

Reply:
oh yeah ethan is right, this is the way to do it

The only thing i don't love about this approach is if we ever add additional parameters we have to remember that `pattern` must remain the last parameter in order for this to keep working (i lost a day debugging this with another function early in my arkouda development). So we should add a comment on the client and server saying that `pattern` should be always be the last parameter since it can contain spaces

Reply:
This doesn't work when the first character of the argument is a space, so I changed the python code to change  the position of the space to the last character in the argument if it was originally in the first position.

Reply:
We need to investigate the first character being a space issue further to see if any other methods are affected, and update them accordingly (or at least denote it with a comment). I think the right answer for all this is to increase our efforts in moving our message processing to JSON

Reply:
I believe I tracked down the issue with the first char being a space. Per the Chapel docs on `.split`, [here](https://chapel-lang.org/docs/builtins/String.html?highlight=split#String.string.split), the split is being done on runs of whitespace when only the number of chunks is supplied. However, if you look at the function allowing the separation string to be provided, [here](https://github.com/Bears-R-Us/arkouda/blob/40e16f155a67e006c33fe0f44100bcb878edd443/src/ServerConfig.chpl#L240), you can specify the single space as the separator. 

As a result, I think we should update the message split to something like:
```
var (objtype, name, chars) = payload.splitMsgToTuple(" ", 3);
```

This will then allow for 
```
        if chars:
            chars = chars[1:] + " " if chars[0] == " " else chars
```
to be removed.

Reply:
Updated msg split and removed the if.